### DISCUSSION TITLE
How to show the delta in read mode, just reading it without cursor being on or keyboard popping on andoid mobile

### DISCUSSION THREAD

**Discussion by daveragos** [singerdmx/flutter-quill#2339] [Mobile]
i was writing a blog app and where the write write and if they wanted to preview the writing so far, i want to show the the written blog in a non edittable way

**Comment by CatHood0** [comment]
You can use [Flutter Quill Easy Parser](https://pub.dev/packages/flutter_quill_delta_easy_parser) which will allow you to convert any paragraph into easy-to-understand paragraphs and will allow you to format them easily. 

I give you a simple example: 

Suppose I have a `Delta` of this style: 

```dart 
[
 {"insert":"Hello "}, 
 {"insert":"this is an example","attributes":{"bold":true}}, 
 {"insert":"Header example"}, 
 {"insert":"\n","attributes":{"header":1}},
 {"insert":"\n"} 
] 
``` 

This itself will become an object that will come from `Flutter Quill Easy Parser` that will look like this: 

```dart
 // obviously this is a visual representation, it is not the object itself. You will have to check the package to see how your `Delta` will come
Document: [
  // The simple paragraph
  Paragraph: [
       Line: "Hello ", Attributes: null 
       Line: "this is an example", Attributes: {"bold": true} 
  ],
  Type: Inline,
  Block Attributes: null,
  // The header paragraph
  Paragraph: [
       Line: "Header Example", Attributes: null
  ],
  Type: Block,
  Block Attributes: {"header": 1} ,
]
``` 

In any case, this type of object effectively encapsulates all the attributes of a Delta, and will allow you to convert this into Widgets that correctly represent the `Delta` of your blog post.
